HW2/MultirankPalindromeTest.py

```diff
@@ -0,0 +1,11 @@
+num = int(input('Enter number:'))
+test = num
+dopel = 0
+while num / 10 > 0:
+    dopel += num % 10
+    num //= 10
+    dopel *= 10
+if test * 10 == dopel:
```

Comment:
Why do we need to multiply test? 
Please rework it a bit in a way we shouldn't multiply input data